### DISCUSSION TITLE
add closed caption to Japan channels?

### DISCUSSION THREAD

**Discussion by ChiFungLam** [iptv-org/community#301] [Q&A] [answered]
Hello. can Japan channel incorporate cc (closed caption) as well? seems that only NHK's Cc is incorporated now, but other channels not e.g. Fuji TV. Its much better to have Cc for a non native Japanese speaker, thanks.

Regds
LAM Chi-fung

**Comment by BellezaEmporium** [comment, marked as answer]
We do not have control over the channels, it is up to the servers + hosts to give CC to those channels. Sadly, only NHK World seems to come from an official source and hence, has CC enabled. Some other official sources don't have CC because they're not aimed to an international community.
